cookies/resources/set-cookie.py

```diff
@@ -20,10 +16,14 @@ def main(request, response):
     < Date: Tue, 04 Oct 2016 18:16:06 GMT
     < Content-Length: 80
     """
-    params = urlparse.parse_qs(request.url_parts.query)
+
+    name = request.GET[b'name']
+    path = request.GET[b'path']
+    cookie = name + b"=1; " + b"Path=" +  path + b"; " + b"Expires=Wed, 09 Jun 2021 10:18:14 GMT"
```

Comment:
Yoo can still use a format string with bytes in both Python 2 and 3

Reply:
Is it okay to explain a bit more what you mean here? Thank you.

The code here was refactored as I wasn't sure what encoding scheme to use to convert string to bytes after the .format() called in the original code.

Reply:
`b"%b=1; Path=%b; Expires=Wed, 09 Jun 2021 10:18:14 GMT" % (name, path)` works, I believe

Reply:
`b"%s=1; Path=%s; Expires=Wed, 09 Jun 2021 10:18:14 GMT" % (name, path)` does work in both. According to [PEP 461](https://www.python.org/dev/peps/pep-0461/) though, 

> %s is included as a synonym for %b for the sole purpose of making 2/3 code bases easier to maintain. Python 3 only code should use %b.

Reply:
```suggestion
    cookie = b"%s=1; Path=%s; Expires=Wed, 09 Jun 2021 10:18:14 GMT" % (name, path)
```